packages/@sanity/cli/src/actions/init-project/initProject.js

```diff
@@ -31,8 +31,10 @@ export default async function initSanity(args, context) {
   const unattended = cliFlags.y || cliFlags.yes
   const print = unattended ? noop : output.print
 
-  print('This utility walks you through creating a Sanity installation.')
-  print('Press ^C at any time to quit.\n')
+  print(`This utility walks you through creating a Sanity project.
```

Comment:
If you actually want these to be on separate lines, it's better to split this up into separate print statements, to maintain proper indentation when printed.